infoschema/tables_test.go

```diff
@@ -1096,6 +1096,52 @@ func (s *testClusterTableSuite) TestSelectClusterTable(c *C) {
 	}
 }
 
+func (s *testClusterTableSuite) TestSelectClusterTablePrivelege(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	slowLogFileName := "tidb-slow.log"
+	f, err := os.OpenFile(slowLogFileName, os.O_CREATE|os.O_WRONLY, 0644)
+	c.Assert(err, IsNil)
+	_, err = f.Write([]byte(
+		`# Time: 2019-02-12T19:33:57.571953+08:00
+# User: user2@127.0.0.1
+select * from t2;
+# Time: 2019-02-12T19:33:56.571953+08:00
+# User: user1@127.0.0.1
+select * from t1;
+# Time: 2019-02-12T19:33:58.571953+08:00
+# User: user2@127.0.0.1
+select * from t3;
+# Time: 2019-02-12T19:33:59.571953+08:00
+select * from t3;
+`))
+	c.Assert(f.Sync(), IsNil)
+	c.Assert(err, IsNil)
+	defer os.Remove(slowLogFileName)
+	tk.MustExec("use information_schema")
+	tk.MustQuery("select count(*) from `CLUSTER_SLOW_QUERY`").Check(testkit.Rows("4"))
+	tk.MustQuery("select count(*) from `CLUSTER_PROCESSLIST`").Check(testkit.Rows("1"))
```

Comment:
Add test for `CLUSTER_PROCESSLIST ` to compare with root by a new user.

Reply:
It's a little difficult add test for `CLUSTER_PROCESSLIST`, So I add a manual test in the PR description 😢